### PR TITLE
[FEATURE] Upgrade to Webpack 4.

### PR DESCRIPTION
**Pull request by Kuzirashi**
Webpack 3 results (full log - https://gist.github.com/Kuzirashi/a642080c30a7693fbb811bc65b04bb20):

Production build:
- app.bundle.js - 2.4 MB
- worker.js - 1.15 MB
- Time: 60.04s

-----------------------------------------------------------------------------------------------------

Webpack 4 results (full log - https://gist.github.com/Kuzirashi/d0d716176c328da61398ed04e7d054e8):

Production build:
- app.bundle.js 2.28 MiB
- worker.js 1.1 MB 
- Time: 64.85s

-----------------------------------------------------------------------------------------------------

Unfortunately HotModuleReplacement in conjunction with worker-loader doesn't work until module type: 'universal' isn't implemented.

- https://github.com/webpack/webpack/issues/6642 - "Webpack 4.0.1 | WebWorker `window is not defined`"
- https://github.com/webpack/webpack/issues/6525 - "Add `target: 'universal'`"

There's bigger issue with CSS order:
- https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/737

### CODE REVIEW

File: app/components/ScheduleWizard/ScheduleWizard.js [hunk 1]
```diff
@@ -190,7 +190,10 @@ class ScheduleWizard extends Component {
       showNotification('Transaction cancelled by the user.', 'danger', 4000);
     }
 
-    this.scheduleBtn.innerHTML = 'Schedule';
+    if (this.scheduleBtn) {
```
Comment (adibas03) [inline app/components/ScheduleWizard/ScheduleWizard.js]: Can you make this a separate PR , since it solves a different issue, and is a ready solution as against the Webpack

Comment (Bagaric) [reply app/components/ScheduleWizard/ScheduleWizard.js]: A fix to this has already been merged into develop in https://github.com/chronologic/eth-alarm-clock-dapp/pull/197.

**Comment by Kuzirashi** [comment]
https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/745#issuecomment-374548449

**Comment by Kuzirashi** [comment]
Replaced extract text plugin with mini-css-extract-plugin (which is now officially supported for Webpack 4), but CSS reloading is not working because of:
- https://github.com/webpack-contrib/mini-css-extract-plugin/issues/31
- https://github.com/webpack-contrib/mini-css-extract-plugin/issues/23

For now it works only after restarting webpack build process.

**Comment by Kuzirashi** [comment]
Now it's better. There's no Hot Module Replacement working yet, but it's definitely possible to use Webpack 4 now. I still have to test using worker though.
